docs/src/content/docs/manual-setup.mdx

```diff
@@ -125,4 +125,7 @@ In the future, we plan to support this use case better to avoid the need for the
 
 ### Use Starlight with SSR
 
-Currently, Starlight does not support [SSR deployment](https://docs.astro.build/en/guides/server-side-rendering/) using Astro’s server adapters. We hope to be able to support this soon.
+Currently, Starlight does not support [Server-Side rendering](https://docs.astro.build/en/guides/server-side-rendering/) for its pages, but you can use Starlight alongside SSR pages in your project. Starlight pages will always be prerendered regardless of your project's output mode.
+
+We hope to be able to support SSR for Starlight pages soon.
```

Comment:
In the knowledge that people often stop reading quickly, let’s flip this to not say “does not support” right away 😁

```suggestion
You can use Starlight alongside custom on-demand rendered pages in your project by following the [“On-demand Rendering Adapters”](https://docs.astro.build/en/guides/server-side-rendering/) guide in Astro’s docs.

Currently, documentation pages generated by Starlight are always prerendered regardless of your project's output mode. We hope to be able to support on-demand rendering for Starlight pages soon.
```